### DISCUSSION TITLE
How to get the selected item from a dropdown in Taiko?

### DISCUSSION THREAD

**Discussion by Ananda0488** [getgauge/taiko#2102] [Development]
In my web page, in Country Name dropdown, the selected country name is 'India'. What will be the code in Taiko, to get the selected value of the dropdown (in this case 'India').

**Comment by zabil** [comment]
There are a few ways to get this value. You can find a few examples here
https://docs.taiko.dev/api/dropdown/

**Reply by Ananda0488** [reply]
Hi Zabil,
Following is the selected option in the dropdown:

![image](https://user-images.githubusercontent.com/85232144/120496840-89145f80-c3db-11eb-8f34-a57bde3bd77e.png)

On doing the following:
await dropDown('Country Name').value()

I am getting: 123
but, I need the output: India

is there any way i can get India?

**Reply by zabil** [reply]
I don't think there's an api for that at the moment. The following returns all the options.

```
await dropDown('Country Name').text() // newline separated
```

You could use the [evaluate](https://docs.taiko.dev/running_scripts_inside_the_browser/) api to return the selected value using javascript on the browser.
